pkg/apis/core/v1alpha1/uisession_types.go

```diff
@@ -107,6 +107,57 @@ func (in *UISessionList) GetListMeta() *metav1.ListMeta {
 
 // UISessionStatus defines the observed state of UISession
 type UISessionStatus struct {
+	// FeatureFlags reports a list of experimental features that have been
+	// enabled.
+	FeatureFlags []UIFeatureFlag `json:"featureFlags" protobuf:"bytes,1,rep,name=featureFlags"`
+
+	// NeedsAnalyticsNudge reports whether the UI hasn't opted in or out
+	// of analytics, and the UI should nudge them to do so.
+	NeedsAnalyticsNudge bool `json:"needsAnalyticsNudge" protobuf:"varint,2,opt,name=needsAnalyticsNudge"`
+
+	// RunningTiltBuild reports the currently running version of tilt
+	// that this UI is talking to.
+	RunningTiltBuild TiltBuild `json:"runningTiltBuild" protobuf:"bytes,3,opt,name=runningTiltBuild"`
+
+	// SuggestedTiltVersion tells the UI the recommended version for this
+	// user. If the version is different than what's running, the UI
+	// may display a prompt to upgrade.
+	SuggestedTiltVersion string `json:"suggestedTiltVersion" protobuf:"bytes,4,opt,name=suggestedTiltVersion"`
+
+	// VersionSettings indicates whether version updates have been enabled/disabled
+	// from the Tiltfile.
+	VersionSettings VersionSettings `json:"versionSettings" protobuf:"bytes,12,opt,name=versionSettings"`
+
+	// TiltCloudUsername reports the username if the user is signed into
+	// TiltCloud.
+	TiltCloudUsername string `json:"tiltCloudUsername" protobuf:"bytes,5,opt,name=tiltCloudUsername"`
+
+	// TiltCloudUsername reports the human-readable team name if the user is
+	// signed into TiltCloud and the Tiltfile declares a team.
+	TiltCloudTeamName string `json:"tiltCloudTeamName" protobuf:"bytes,6,opt,name=tiltCloudTeamName"`
+
+	// TiltCloudSchemeHost reports the base URL of the Tilt Cloud instance
+	// associated with this Tilt binary. Usually https://cloud.tilt.dev
```

Comment:
```suggestion
	// associated with this Tilt process. Usually https://cloud.tilt.dev
```

Reply:
(this is mostly changed via env var when running tilt)

Reply:
done